libs/blocks/library-config/lists/blocks.js

```diff
@@ -110,7 +110,9 @@ export default async function loadBlocks(blocks, list, query) {
       name.textContent = getAuthorName(pageBlock) || getBlockName(pageBlock);
       const copy = document.createElement('button');
       copy.addEventListener('click', (e) => {
-        const table = getTable(pageBlock, getBlockName(pageBlock), block.path);
+        const table = pageBlock.className === 'section-metadata'
+          ? `${getTable(pageBlock, getBlockName(pageBlock), block.path)} ---`
```

Comment:
Nit: store the result of `getTable(pageBlock, getBlockName(pageBlock), block.path)` in a variable and reuse that.